nservicebus/messaging/callbacks_enum-reply_callbacks_[2,).partial.md

```diff
@@ -0,0 +1,5 @@
+NOTE: When replying with an enum value, the replying endpoint also needs to reference the `NServiceBus.Callbacks` package.
+
+If the endpoint only replies to callbacks use the reply-only mode as shown below:
```

Comment:
you call it 'reply-only' mode, but you named the config option `makesRequests` which doesn't align very well imho (I prefer 'reply-only')